Cargo.toml

```diff
@@ -26,9 +27,9 @@ prettytable-rs = "0.8.0"
 regex = "1"
 reqwest = {version = "0.10", features = ["native-tls-vendored", "blocking", "json", "gzip"]}
 rpassword = "4"
-rusqlite = {version = "0.21", features = ["bundled"]}
```

Comment:
What are the implications of removing this? I believe it may require that users have a C compiler and/or other tools like Cmake installed; I'm not opposed to this but we should figure out why we were using it before removing it!